### PR TITLE
new subpkg: add gator cli

### PR DESCRIPTION
**Pull request by vanlongme**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: gatekeeper-3.14.yaml [hunk 1]
```diff
@@ -48,6 +49,14 @@ subpackages:
...
       provides:
         - gatekeeper-compat=${{package.full-version}}
+  - name: ${{package.name}}-gator
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}
+          cp bin/gator ${{targets.subpkgdir}}/gator
+    dependencies:
+      provides:
+        - gatekeeper-compat=${{package.full-version}}
```
Comment (ajayk) [inline gatekeeper-3.14.yaml:59]:
```suggestion
        - gatekeeper-gator=${{package.full-version}}
```

Comment (vanlongme) [reply gatekeeper-3.14.yaml]: thanks @ajayk

**Comment by tuananh** [comment]
@vanlongme update other version stream as well? we have gatekeeper-3.12, gatekeeper-3.13 & gatekeeper-3.14 here.

**Comment by vanlongme** [comment]
> @vanlongme update other version stream as well? we have gatekeeper-3.12, gatekeeper-3.13 & gatekeeper-3.14 here.

3.12 and 3.13 have a lot of  CVEs, so I don't really want to keep track on it

**Comment by tuananh** [comment]
@ajayk can you take a look at this?

**Comment by vanlongme** [comment]
@ajayk Do you think that update action have some issue ?

File: gatekeeper-3.14.yaml [hunk 2]
```diff
@@ -49,6 +50,15 @@ subpackages:
       provides:
         - gatekeeper-compat=${{package.full-version}}
 
+  - name: ${{package.name}}-gator
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}
+          cp bin/gator ${{targets.subpkgdir}}/gator
```
Comment (ajayk) [inline gatekeeper-3.14.yaml:57]: Can you use mv instead of cp and tests

**Review by ajayk — CHANGES_REQUESTED** [review]

File: gatekeeper-3.14.yaml (hunk 2 above)
Comment (vanlongme) [reply gatekeeper-3.14.yaml]: I updated and tested on my machine.

**Review by ajayk — APPROVED** [review]

**Comment by tuananh** [comment]
@ajayk looks like CI failure when uploading pkgs https://github.com/wolfi-dev/os/actions/runs/8700106538/job/23859816546

**Pull request merged**